components/scream/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -728,7 +728,7 @@ void RRTMGPRadiation::run_impl (const int dt) {
     // Compute band-by-band surface_albedos. This is needed since
     // the AD passes broadband albedos, but rrtmgp require band-by-band.
     rrtmgp::compute_band_by_band_surface_albedos(
-      ncol, m_nswbands,
+      ncol, nswbands,
```

Comment:
This is fine, but FYI, the `m_lay` issue was b/c you used `this` inside a KOKKOS_LAMBDA. It's ok to use `m_xyz` here, since you're not _inside_ a `__device__ [=]` kind of lambda.

Reply:
True, this just ended up getting changed because I found a case of using `m_nswbands` inside a YAKL device lambda. Would it be better to use the `m_nswbands` version here? I think it probably doesn't matter I was just trying to be consistent in my use...